CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Fix courses badges css.
```

Comment:
In your commit message :
```diff
- On the CourseProductItem, when user tries to enroll or enunroll to a course run,
+ On the CourseProductItem, when user tries to enroll or unroll to a course run,

```

Then add in the commit footer
`Close #1858`

Reply:
oops